src/notation/view/notationviewinputcontroller.cpp

```diff
@@ -343,6 +436,9 @@ void NotationViewInputController::mousePressEvent(QMouseEvent* event)
             selectType = SelectType::ADD;
         }
         viewInteraction()->select({ hitElement }, selectType, hitStaffIndex);
+    } else if (hitElement && event->button() == Qt::MouseButton::LeftButton
+               && event->modifiers() == Qt::KeyboardModifier::ControlModifier) {
+        viewInteraction()->select({ hitElement }, SelectType::ADD, hitStaffIndex);
```

Comment:
restores MU3 behaviour of toggling selection for an element on left button+CTRL click